2025年昇腾AI创新大赛-昇思模型开发挑战赛（S1赛季)--MultiModal赛题--ghahqj队提交

ghahqj opened the pull request.

ghahqj committed:
1

ghahqj committed:
2

ghahqj committed:
Add files via upload

qhzhuang commented on 2025-Ascend-Innovation-Contest/S1/MultiModal/ghahqj/README.md:
```diff
@@ -0,0 +1,64 @@
+
+1. **RoPE 算子融合**：使用 MindSpore 特有的 `ops.rotary_position_embedding` 融合算子替代原本的手动实现，大幅减少算子调用开销并提升计算密度。 
+2. **算子替换**：将 `swapaxes` 替换为 `ops.transpose`，将切片操作替换为 `ops.split`，获得更好的图编译性能。
+3. **移除冗余 Cast**：移除 Softmax 前后不必要的 `float32` 类型转换，直接在当前精度下计算。
```
这么做是不对的

ghahqj replied:
已经修改了

qhzhuang reviewed (changes requested):
你多提交了一个文件 er.name.ghahqj

ghahqj committed:
Delete er.name ghahqj

ghahqj committed:
Remove redundant float32 casts in README

Removed unnecessary float32 type conversions before and after Softmax to streamline calculations.
